contracts/PartyBid.sol

```diff
@@ -50,7 +57,13 @@ contract PartyBid is Party {
 
     event Bid(uint256 amount);
 
-    event Finalized(PartyStatus result, uint256 totalSpent, uint256 fee, uint256 totalContributed);
+    // @notice emitted when a party is won, lost, or expires.
+    // @param result The `WON` or `LOST` final party state.
+    // @param totalSpent The amount of eth actually spent, including the price of the NFT and the fee.
+    // @param fee The eth fee paid to PartyDAO.
+    // @param totalContributed Total eth deposited by all contributors, including eth not used in purchase.
+    // @param expired True if the party expired before reaching a reserve / placing a bid.
+    event Finalized(PartyStatus result, uint256 totalSpent, uint256 fee, uint256 totalContributed, bool expired);
```

Comment:
i believe @steveklebanoff in the past has asked me not to change the interface fo this event - steve, wdyt?

Reply:
our new infra is more flexible, modifying this interface is OK with me 👍